src/status_im2/contexts/share/events.cljs

```diff
@@ -0,0 +1,28 @@
+(ns status-im2.contexts.share.events
```

Comment:
@flexsurfer @erikseppanen @ilmotta I think we need to define what a context is. imo this share feature is not a "context" and now we are just starting to run into the same issue as the previous implementation of status_im codebase where there is a long list of shallow folders.

As the pr description says, this is for shell profile only. So either there is a Profile context (which has/will have lots of pieces to it) or the shell context. Either of these two would be better imo.

Reply:
This probably belongs inside shell but I wasn't 100% sure at the moment of creating this component.
I'll talk with @Parveshdhull about this as well since he wrote most of shell.

Reply:
Hi @siddarthkay, As this share screen is not shell specific it can also be opened from the home screen top bar, so not sure if shell namespace is the right place for it.
Top-nav itself is inside `status-im2.common.home`, so maybe we can create `events.cljs` in that namespace.

Reply:
something like `status-im2.common.home.share`?

Reply:
btw, I am not just referring to the events file, this is also relevant for the view and style too. Also imo the view and style should be in their own folder too.
e.g `context.share.share.view`, `context.share.share.style` `context.share.events`

Reply:
Cool, I can kick off this discussion @ilmotta. 👍